Assets/Scripts/HUD/ProgressBar.cs

```diff
@@ -5,39 +5,49 @@
 public class ProgressBar : MonoBehaviour
 {
     private Slider slider;
+    public Gradient gradient;
+    public Image fill;
 
 
     private void Awake()
     {
         slider = gameObject.GetComponent<Slider>();
         slider.maxValue = 100f;
         slider.minValue = 0f;
-        slider.value = 75f;
+        slider.value =5f;
+        fill.color = gradient.Evaluate(0.1f);
```

Comment:
Just for tidiness purposes you can put this in like a SliderInit() method, also if interested, research using TryGetComponent